tests/functional/test_init.py

```diff
@@ -37,3 +39,11 @@ def test_version_option():
     assert not ret.stderr, "Nothing should be written to stderr"
     assert ret.returncode == 0, "A non-successful return code was provided"
     assert ret.stdout == expected_output, "Output did not match expected input"
+
+
+def test_phylum_pubkey_is_constant(tmp_path):
+    """Ensure the RSA public key in use by Phylum has not changed."""
+    phylum_pubkey_url = "https://raw.githubusercontent.com/phylum-dev/cli/main/scripts/signing-key.pub"
+    downloaded_key_path: Path = tmp_path / "signing-key.pub"
```

Comment:
Just out of curiosity, is this how you concatenate paths in Python? Because using division for that feels very wrong.

Reply:
Yes...this is the new standard/preferred way to concatenate paths. It used to be done with `os.path.join()` calls, but now the `pathlib` module in the standard library has taken over as the "pythonic" way.
Changing felt odd at first but now it is second nature!